test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -25,11 +25,11 @@ namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Audit
     /// Provides Audit specific tests.
     /// </summary
     [HttpIntegrationFixtureArgumentSets(DataStore.CosmosDb, Format.Json)]
-    public class AuditTests : IClassFixture<AuditTestFixture>
+    public partial class AuditTests : IClassFixture<AuditTestFixture>
```

Comment:
Per the refactoring done by Brandon, we don't need AuditTests as partial class for batch. Do we have good reason to create partial class for transaction? And I don't see the other part of partial class AuditTests.

Reply:
Yes I saw the refactoring and removed the two partial classes from my PR as well. This one I missed to remove. Thank you for pointing it out.